src/languages/en.ts

```diff
@@ -3466,7 +3466,7 @@ export default {
         roomNameInvalidError: 'Room names can only include lowercase letters, numbers, and hyphens.',
         pleaseEnterRoomName: 'Please enter a room name.',
         pleaseSelectWorkspace: 'Please select a workspace.',
-        renamedRoomAction: ({oldName, newName}: RenamedRoomActionParams) => ` renamed this room from ${oldName} to ${newName}`,
```

Comment:
Why did we change this Text? If required shouldn't we also need to change the translations @neonbhai

Reply:
We needed to remove the leading word from earlier format, consequently we have to remove the leading space from `renamedRoomAction`. Confirming `es` translation on Slack [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1721836752589369)